pkg/asset/manifests/tectonic.go

```diff
@@ -54,6 +62,13 @@ func (t *Tectonic) Generate(dependencies asset.Parents) error {
 	worker := &machines.Worker{}
 	master := &machines.Master{}
 	dependencies.Get(installConfig, clusterk8sio, worker, master)
+	kubeadminPassword, kubeadminPasswordHash, err := generateRandomPasswordHash()
```

Comment:
Why are we generating this instead of asking the user if they want to supply a password?

Reply:
this (really more this: https://github.com/openshift/origin/pull/21580)  creates an initial pre-identity-provider-setup user that can access registry, prometheus, web-console, grafana, while at the same time _not_ creating a bunch of roles/identities/objects that have to be cleaned up once an admin does set up an idp.  @enj definitely does not want to let a user choose this password, it has to be secure.

Reply:
> @enj definitely does not want to let a user choose this password, it has to be secure.

14 random chars is less seceure than my default ;).  And I'd really like to not take responsibility for this :p.  Can it be a day-2 operation?